### PR TITLE
3rd change

### PR DESCRIPTION
**Pull request by mocadev1**

### CODE REVIEW

File: src/font.md [hunk 1]
```diff
@@ -3,3 +3,5 @@ ___
 The preferred family font for the project would be JetBrains Mono, but if not available we could also use Cascadia Code
 
 Did some changes just to that it seems that it attended the review xd
+
+But again, I'm adding something.
```
Comment (alexandergmzx) [inline src/font.md]: Maybe its not necessary, you should add some other libraries.

File: src/js.md [hunk 2]
```diff
@@ -2,3 +2,5 @@
 
 In the front-end we'll be using react with **functional components**.
 For the back-end we'll be using express.
+
+A new line.
```
Comment (alexandergmzx) [inline src/js.md]: Also not needed.

**Review by alexandergmzx — CHANGES_REQUESTED** [review]
Erase or submit other changes

File: src/font.md (hunk 1 above)
Comment (Israel-SanchezH) [reply src/font.md]: Looks good!!!

File: src/js.md (hunk 2 above)
Comment (Israel-SanchezH) [reply src/js.md]: Please add the CSS file

**Review by Israel-SanchezH — CHANGES_REQUESTED** [review]

**Review by alexandergmzx — COMMENTED** [review]
Well done!

**Pull request merged**